src/sbuf.c

```diff
@@ -1235,6 +1236,44 @@ static bool setup_tls(struct tls_config *conf, const char *pfx, int sslmode,
 	return true;
 }
 
+static bool tls_config_changed(struct tls_config *new_server_connect_conf)
+{
+	return (strings_equal(new_server_connect_conf->ca_file, server_connect_conf->ca_file) &&
+		strings_equal(new_server_connect_conf->ca_path, server_connect_conf->ca_path) &&
```

Comment:
There's a nasty problem here with this comparison. It seems likely that for rotations people might change the contents cert/key while keeping the path the same. This comparison wouldn't detect that. I don't know if we can actually compare this (especially when considering race conditions). So I think this limitation is probably fine, but we should document this in the user facing documentation. So for the `server_tls_ca_file` and `client_tls_ca_file` docs add something like this:
> If the contents of this file are changed without changing the actual setting, the new CA file will be used for new connections. Existing connections won't be closed though. If it's necessary for security reasons that connections using the old CA are recycled ASAP, it's advised to run `RECONNECT` after the reload.

Reply:
This only checks the `server_tls_ca_file`, right?

Reply:
Ahh yep both are impacted by less aggressive connection recycling on `RELOAD` 11f099b